src/match.scm

```diff
@@ -190,58 +190,58 @@
   (if (or (not (pair? expr)) (eq? (car expr) 'quote))
       expr
       (let ((enew (apply-patterns plist expr)))
-	(if (eq? enew expr)
-	    ;; expr didn't change; move to subexpressions
-	    (map (lambda (subex)
-		   (if (not (pair? subex))
-		       subex
-		       (pattern-replace plist subex)))
-		 expr)
-	    enew))))
+        (if (eq? enew expr)
+            ;; expr didn't change; move to subexpressions
+            (map (lambda (subex)
+                   (if (not (pair? subex))
+                       subex
+                       (pattern-replace plist subex)))
+                 expr)
+            enew))))
 
 (define-macro (pattern-set . pats)
   ; (pattern-lambda (x ...) ...) => x
   (define (pl-head p) (car (cadr p)))
   (receive
    (pls others) (separate (lambda (x)
-			    (and (pair? x) (length= x 3)
-				 (eq? (car x) 'pattern-lambda)
-				 (pair? (cadr x))))
-			  pats)
+                            (and (pair? x) (length= x 3)
+                                 (eq? (car x) 'pattern-lambda)
+                                 (pair? (cadr x))))
+                          pats)
    (let ((heads (delete-duplicates (map pl-head pls)))
-	 (ht    (gensym)))
+         (ht    (gensym)))
      `(let ((,ht (make-table)))
-	,@(map (lambda (h)
-		 `(table-set! ,ht ',h (list
-				       ,@(filter (lambda (p)
-						   (eq? (pl-head p) h))
-						 pls))))
-	       heads)
-	(vector 'pattern-set ,ht (list ,@others))))))
+        ,@(map (lambda (h)
+                 `(table-set! ,ht ',h (list
+                                       ,@(filter (lambda (p)
+                                                   (eq? (pl-head p) h))
+                                                 pls))))
+               heads)
+        (vector 'pattern-set ,ht (list ,@others))))))
 
 (define (plambda-expansion pat expr expander args)
   (let ((m (match pat expr)))
     (if m
-	(apply expander (map (lambda (var) (cdr (or (assq var m) '(0 . #f))))
-			     args))
-	#f)))
+        (apply expander (map (lambda (var) (cdr (or (assq var m) '(0 . #f))))
+                             args))
+        #f)))
 
 (define-macro (pattern-lambda pat body)
   ; given a pattern p, return the list of capturing variables it uses
   (define (patargs- p)
     (cond ((and (symbol? p)
-		(not (memq p '(_ ...))))
-	   (list p))
+                (not (memq p '(_ ...))))
+           (list p))
 
-	  ((pair? p)
-	   (if (eq? (car p) '-/)
-	       '()
-	       (delete-duplicates (apply append (map patargs- (to-proper (cdr p)))))))
+          ((pair? p)
+           (if (eq? (car p) '-/)
+               '()
+               (delete-duplicates (apply append (map patargs- (to-proper (cdr p)))))))
 
-	  (else '())))
+          (else '())))
   (define (patargs p)
     (cons '__ (patargs- p)))
   (let* ((args (patargs pat))
-	 (expander `(lambda ,args ,body)))
+         (expander `(lambda ,args ,body)))
     `(lambda (__ex__)
        (plambda-expansion ',pat __ex__ ,expander ',args))))
```

Comment:
sorry, yes, i did a commit of just whitespace normalization (https://github.com/JuliaLang/julia/commit/c1ea4d66da2237d7026780d640ba0b0feeca59ec)